data/src/main/java/dev/jdtech/jellyfin/models/VideoMetadata.kt

```diff
@@ -0,0 +1,46 @@
+@file:Suppress("Unused")
+
+package dev.jdtech.jellyfin.models
+
+data class VideoMetadata(
+    val resolution: List<Resolution>,
+    val displayProfiles: List<DisplayProfile>,
+    val audioChannels: List<AudioChannel>,
+    val audioCodecs: List<AudioCodec>,
+    val isAtmos: List<Boolean>
+)
+
+enum class Resolution(val raw: String) {
+    SD("SD"),
+    HD("HD"),
+    UHD("4K");
+}
+
+enum class DisplayProfile(val raw: String) {
+    SDR("SDR"),
+    HDR("HDR"),
+    HDR10("HDR10"),
+    DOLBY_VISION("Vision"),
+    HLG("HLG");
+}
+
+enum class AudioChannel(val raw: String) {
+    CH_2_0("2.0"),
+    CH_2_1("2.1"),
+    CH_5_1("5.1"),
+    CH_7_1("7.1");
+}
+
+enum class AudioCodec(val raw: String) {
+    FLAC("FLAC"),
+    MP3("MP3"),
+    AAC("AAC"),
+    AC3("Digital"),
+    EAC3("DD+"),
+    VORBIS("VORBIS"),
+    DTS("DTS"),
+    TRUEHD("TrueHD"),
+    OPUS("OPUS");
+
+    override fun toString() = this.raw.lowercase()

```

Comment:
Dolby Digital and Digital Plus are never shown. I think it is because you are comparing the codec from Jellyfin with `raw` but you should compare it to the lowercase enum. But maybe then the name `raw` does not make sense anymore? :thinking: 

This works:

```suggestion
    override fun toString() = super.toString().lowercase()

```

Reply:
Yup, I was skeptical about codecs and there is no list specified in the Jellyfin API docs. I think for `Dolby Digital` and `Dolby Digital+` we will have to parse it like we did for `Atmos`? What do you think @jarnedemeulemeester?